apitools/org.eclipse.pde.api.tools.ui/src/org/eclipse/pde/api/tools/ui/internal/completion/APIToolsJavadocCompletionProposalComputer.java

```diff
@@ -377,11 +378,11 @@ public String getErrorMessage() {
 	public void sessionEnded() {
 		if (fImageHandle != null) {
 			fImageHandle.dispose();
+			fImageHandle = null;
```

Comment:
This is not multithreading save, fImageHandle must be extracted into a local variable if it can really be modified / accessed from multiple threads

Reply:
s.o.